runtime/src/main/java/org/corfudb/runtime/object/transactions/AbstractTransactionalContext.java

```diff
@@ -122,7 +121,7 @@
     private final WriteSetInfo writeSetInfo = new WriteSetInfo();
 
     @Getter
-    private final ReadSetInfo readSetInfo = new ReadSetInfo();
+    private final ConflictSetInfo readSetInfo = new ConflictSetInfo();
```

Comment:
In the case of write-write transactions, I don't think "ReadSetInfo" is even necessary. It happens that AbstractTransactionContext keeps track of them. AccessSetInfo isn't very descriptive.

I still believe ConflictSet best describes what it is.